packages/dom-utilities/.eslintrc.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/react"],
```

Should probably use `react--legacy` here since this is moving old code which uses `I` prefixes on interface names.
```suggestion
  "extends": ["plugin:@fluentui/eslint-plugin/react--legacy"],
```